spec/image_builder_spec.rb

```diff
@@ -80,16 +80,14 @@
 
       # Build
       expect(builder).to receive(:system)
-        .with({"AYTESTS_FILES_DIR" => files_dir.to_s},
+        .with({"AYTESTS_FILES_DIR" => files_dir.to_s, "AYTESTS_LINUXRC" => "vnc=1",
+               "AYTESTS_PROVIDER" => provider.to_s, "AYTESTS_WEBSERVER_PORT" => "8888"},
```

Comment:
To be honest, this doesn't look like the most explicitly declarative spec ever.  Would it be possible to have things like

```
it "adds the variables read from the linuxrc file" do
  #some expect
end
```
instead of a "reimplementation" of the steps were such checks are simply implicit?